scripts/src/main/resources/scripts/command/docker

```diff
@@ -105,11 +101,44 @@ function doSetup() {
         doRunCommand "cp -a '${mount_dir}'/*.app /Applications/"
         doRunCommand "hdiutil detach -force '${mount_dir}'"
       fi
-      read -r -p "Docker Desktop has been installed. Please read https://github.com/devonfw/ide/blob/master/documentation/docker.asciidoc#setup for enabling kubernetes and further important details. Press enter to continue" answer
+      read -r -p "Docker Desktop has been installed. Please read https://github.com/devonfw/ide/blob/master/documentation/docker.asciidoc#setup for enabling kubernetes and further important details. Press enter to continue" answer     
+      kill -9 $PPID
```

Comment:
kill -9 $PPID will kill the parent process what is the shell.
This looks kind of dangerous and hackish. I am not sure if this is the way to go...